scripts/apps/workspace/content/views/profile-settings.html

```diff
@@ -33,13 +33,15 @@ <h2 class="sd-page__page-heading" translate>Content Profiles</h2>
                     ng-repeat="type in ctrl.items| orderBy: 'label' | filter: query track by type._id"
                     ng-class="{inactive: !type.enabled}"
                     ng-show="(!active_only || (active_only && type.enabled)) && (contentTypeFilter == null || contentTypeFilter === type.type)"
+                    data-test-id="content-profile"
+                    data-test-value="{{ type.label }}"
                 >
                     <div class="card-box__header">
                         <div class="dropdown" dropdown>
-                            <button class="dropdown__toggle" dropdown__toggle>
+                            <button class="dropdown__toggle" dropdown__toggle data-test-id="content-profile-actions">
                                 <i class="icon-dots-vertical"></i>
                             </button>
-                            <ul class="dropdown__menu more-activity-menu pull-right">
+                            <ul class="dropdown__menu more-activity-menu pull-right" data-test-id="actions--options">
```

Comment:
keep the same beginning of data-test-id tag when they're related  - 
`content-profile-actions` / `content-profile-actions--options`